### PR TITLE
Rich email notifications

### LINKED ISSUE #11: Student: Notification about package
### What I'm trying to achieve (plain english)
Send a rich notification to a student when a package has arrived for them. This should include location, tracking number, and size. 

### As a...
Student

### I want...
to receive rich notifications about my package arrival

### So that...
I know what packages have arrived

(Does finished issue complete this checklist?)

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).

### PR DESCRIPTION
**Pull request by luftdanmark**
Adds automated email notifications upon check-in of a package.

✅ Mobile friendly & responsive
✅ Plaintext alternative included for non HTML email clients

screen shot 2018-02-25 at 16 46 51

Closes #11

### CODE REVIEW

File: mailmule/checkin/views.py [hunk 1]
```diff
@@ -112,6 +113,9 @@ def package_checkin_form_view(request, user_id):
                 return render(request, 'checkin/main.html', ctx, status=400)
 
             package.save()
+
+            send_package_arrival_email(package)
```
Comment (WalkerGriggs) [inline mailmule/checkin/views.py]: Can we get an error response if this email fails to send? Does msg.send() return an error to check?

File: mailmule/users/management/commands/populate_db.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def _create_student(self):
 
         user = User.objects.create_user(**details)
 
-        EmailAddress.objects.create(user=user, email='john@student.com', verified=True, primary=True)
+        EmailAddress.objects.create(user=user, email='john@student.com', verified=True).set_as_primary()
```
Comment (WalkerGriggs) [inline mailmule/users/management/commands/populate_db.py]: Do we have secondary emails implemented? What is the used of a secondary email? Account confirmation?

Comment (luftdanmark) [reply mailmule/users/management/commands/populate_db.py]: Yeah, we've got secondary emails set up as part of django.contrib.auth. Just allows you to link multiple emails to an account. (We only send mail to the primary)

File: mailmule/checkin/views.py (hunk 1 above)
Comment (luftdanmark) [reply mailmule/checkin/views.py]: If you look at /checkin/utils.py, we use django's send_mail, which calls the chosen email backend (in dev: we spoof a local smtp server, in prod: we call the mailgun API). Exceptions and errors are caught by django and failed calls will be automatically retried by the backends.

Comment (luftdanmark) [reply mailmule/checkin/views.py]: In other words, the user shouldn't be exposed to any errors.

**Comment by luftdanmark** [comment]
@WalkerGriggs, see my comments and merge at will 👍 💯

**Comment by WalkerGriggs** [comment]
@luftdanmark, done and done

**Pull request merged**